media-libs/gexiv2/gexiv2-0.10.10.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit meson python-r1 toolchain-funcs vala xdg-utils
+
+DESCRIPTION="GObject-based wrapper around the Exiv2 library"
+HOMEPAGE="https://wiki.gnome.org/Projects/gexiv2"
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://gitlab.gnome.org/GNOME/gexiv2.git"
+	inherit git-r3
+else
+	SRC_URI="mirror://gnome/sources/${PN}/$(ver_cut 1-2)/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd"
+fi
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+IUSE="introspection python static-libs test vala"
+
+REQUIRED_USE="
+	python? ( introspection ${PYTHON_REQUIRED_USE} )
+	test? ( python introspection )
+	vala? ( introspection )
+"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-libs/glib:2
+	media-gfx/exiv2:=
+	introspection? ( dev-libs/gobject-introspection:= )
+	vala? ( $(vala_depend) )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-libs/gobject-introspection-common
+	virtual/pkgconfig
+	test? (
+		dev-python/pygobject
```

Comment:
Missing SLOT in pygobject depend

Reply:
Will fix.

Reply:
Huh... honestly unsure which slot to use here. Introspection is apparently disabled
for py2, so use the 3 slot?